wagtail/admin/tests/test_rich_text.py

```diff
@@ -89,10 +88,59 @@ def setUp(self):
 
         self.login()
 
+    @override_settings()
```

Comment:
What’s the point of overriding no setting?
Does this restores the `WAGTAILADMIN_RICH_TEXT_EDITORS` setting that we remove inside the method?
If so, adding a comment would make it less weird.

Reply:
Yep, see the example beginning "You can also simulate the absence of a setting..." at https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.modify_settings . Will add a comment...

Reply:
Fixed in 20697186e7611ffa0ae41cb9db8e3daf68dbfd88